src/modules/data/blocks/ticket/sagas.js

```diff
@@ -25,7 +26,7 @@ import { wpREST } from '@moderntribe/common/utils/api';
 import { config, restNonce } from '@moderntribe/common/src/modules/utils/globals';
```

Comment:
We'll probably need to import this like:

`import { globals } from '@moderntribe/common/utils';`